### PR TITLE
Release notes for jpo-s3-deposit 1.5.0

### PR DESCRIPTION
**Pull request by dmccoystephenson**
## Problem
Next month we will be releasing a new version of this project. Release notes need to be prepared.

## Solution
The `Release_notes.md` file has been updated for the 1.5.0 release.

## Dependent PR
Some of the changes mentioned are still in PR review. This PR can be merged after #19.

### CODE REVIEW

File: docs/Release_notes.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
 Jpo-s3-deposit Release Notes
 ----------------------------
 
+Version 1.5.0, released June 2024
+----------------------------------------
+### **Summary**
+The changes for the jpo-s3-deposit 1.5.0 release include updated hard-coded kafka properties to be configurable, added support for GCS BLOB storage & an updated java version for the dev container.
```
Comment (drewjj) [inline docs/Release_notes.md:7]: Lets capitalize Kafka in these notes. There are a few occurrences.

Comment (drewjj) [inline docs/Release_notes.md:7]: I think we also want to capitalize Java.

**Review by drewjj — APPROVED** [review]
Looks good, just some small capitalization things.

File: docs/Release_notes.md (hunk 1 above)
Comment (dmccoystephenson) [reply docs/Release_notes.md]: Capitalized!

Comment (dmccoystephenson) [reply docs/Release_notes.md]: Capitalized!

Comment (dmccoystephenson) [reply docs/Release_notes.md]: Capitalized!

**Review by mwodahl — APPROVED** [review]
Looks good to me!

**Pull request merged**